packages/react-ui-kit/styleguide.config.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+const fs = require('fs');
+const path = require('path');
+const docgen = require('react-docgen');
+const packageJson = require('./package.json');
+
+function resolve(...paths) {
+  return fs.realpathSync(path.join(__dirname, ...paths));
+}
+
+module.exports = {
+  title: `Wireapp React Ui Kit v${packageJson.version}`,
+  skipComponentsWithoutExample: true,
+  sections: [
+    {
+      name: 'Forms',
+      components: 'src/Form/**/*.tsx',
+      ignore: [
+        'src/Form/Input.tsx',
+        'src/Form/ShakeBox.tsx',
+        'src/Form/InputBlock.tsx',
+        'src/Form/InputBlock.tsx',
+        'src/Form/RoundIconButton.tsx',
+        'src/Form/InputSubmitCombo.tsx',
+      ],
+    },
+    {
+      name: 'Layout',
+      components: 'src/Layout/**/*.tsx',
+      ignore: [
+        'src/Layout/headerMenu/HeaderSubMenu.tsx',
+        'src/Layout/headerMenu/MenuContent.tsx',
+        'src/Layout/headerMenu/MenuItems.tsx',
+        'src/Layout/headerMenu/MenuLink.tsx',
+        'src/Layout/headerMenu/MenuOpenButton.tsx',
+        'src/Layout/headerMenu/MenuScrollableItems.tsx',
+        'src/Layout/headerMenu/MenuSubLink.tsx',
+      ],
+    },
+    {name: 'Identity', components: 'src/Identity/**/*.tsx'},
+    {name: 'Menu', components: 'src/Menu/**/*.tsx', ignore: ['src/Menu/MenuModal.tsx']},
+    {name: 'Modal', components: 'src/Modal/**/*.tsx', ignore: ['src/Modal/Overlay.tsx']},
+    {name: 'Misc', components: 'src/Misc/**/*.tsx', ignore: ['src/Misc/IsInViewport.tsx']},
+    {name: 'Typography', components: 'src/Text/**/*.tsx'},
+    {
+      name: 'Icons',
+      components: 'src/Icon/SVGIcon.tsx',
+    },
+    {
+      name: 'Colors',
+      content: 'src/Identity/colors.md',
+    },
+  ],
+  styleguideComponents: {
+    Wrapper: resolve('styleguide/wrapper.js'),
+  },
+  resolver: docgen.resolver.findAllExportedComponentDefinitions,
+  styles: {
```

Comment:
Is this still needed in conjunction with the wrapper?

Reply:
Note: `StyledApp` should already have a background color

Reply:
yes, because it's the style for the box which holds the demo of component, without this that box would look ugly.

Reply:
https://github.com/wireapp/wire-web-packages/pull/2937/commits/6d5601ff2f407e25d5483c20427225c49f9cccb1